perfkitbenchmarker/scripts/step_load_driver/launch_mysql_service.py

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/env python
```

Comment:
script location: does it have to be in a sub-dir under the scripts dir? consider one of these two options:

1. put it directly under scripts dir.
2. rename the "step_load_driver" to database_scripts. We will then put all future database benchmarking related scripts to this dir.